lib/app-router.js

```diff
@@ -84,6 +87,40 @@ module.exports = function (router) {
 
         break;
 
+      case 'apex':
+        let message = 'Apex Scripts run.';
+        async.eachSeries(param, function iteratee(apex, callback) {
+          console.log(`running ${apex}`);
+          script = `${startingDirectory}cd ${directory};export FORCE_SHOW_SPINNER=;sfdx force:apex:execute -f ${apex}`;
+
+          commands.run(command, script, (commandResult, commandErr) => {
+            if (commandErr) {
+              console.log(commandErr);
+              callback(commandErr);
+
+            } else {
+              console.log(commandResult);
+              // doesn't seem to be returning until after everything else runs.  fix UI output
```

Comment:
Yeah, that's because it's async. I think we should change the behavior if it's not outputting correctly.

Reply:
actually, this was fixed by adding in the async from the previous comment--I just forgot to take out my comment.  

Because the yaml lets your have multiple apex scripts to run (we're going to have the same with data import and packages!) we need to have the run in the order specified by the user.  Because the ajax calls were async, the loop needed to be async before returning.  

If that doesn't make sense in theory, it certainly worked in practice.